docs/manual/developer/03_creating_content.md

```diff
@@ -558,6 +558,61 @@ controls:
       and to use the targeted policy.
 ```
 
+### Reporting status
+
+In some cases, it's useful to know the status of a certain control for a
+specific product. In order to better portray this, it's possible to set
+such information on each control using the `status` key.
+
+The `status` key may hold the following values:
+
+* `pending`: The control is not yet evaluated for the product.
+
+* `not applicable`: The control is not applicable to this product.
+
+* `inherently met`: The control is inherently met by the product.
```

Comment:
This is where the `automated` key used to fall short, an explanation in notes is necessary to explain why it is not automated. The`status` key seems to handle this better.

Question about `inherently met`, does this mean that a product satisfies a control by design and no configuration of any sort is needed? That compliance would no be broken by misconfigurations.

Reply:
@yuumasato pretty much. Or that: "misconfiguring this is non-trivial".